projects/module-1/rock-paper-scissors-solo-v2.md

```diff
@@ -59,23 +59,24 @@ In this project, we will not be providing detailed iterations. We want you to ex
 
 ### Architecture
 
-Your entire application will consist of one HTML page. You will have three JavaScript files:
+Your entire application will consist of one HTML file, one CSS file and one JS file.
 
-1. A `player.js` file that contains a `Player` class.
-   * `Player` methods must include, _but are not limited to_:
-     - `constructor` - properties should include: `name` (ex: `'Human'`), `token` (ex: `'👩🏻'`), `wins` (ex: `0`)
+1. A `main.js` file that contains all of your JavaScript logic.
+   * You'll need a `createPlayer` function, that should return an object with the following properties:
+     - `name` (ex: `'Human'`), `token` (ex: `'👩🏻'`), `wins` (ex: `0`)
+
+   * Your player should have access to functions that must include, _but are not limited to_:
      - `saveWinsToStorage` - only necessary if you choose the localStorage extension
      - `retrieveWinsFromStorage` - only necessary if you choose the localStorage extension
      - `takeTurn`
-2. A `game.js` file that contains a Game class.
-   * A `Game` should include:
-     - Two `Player` instances
+   * A `createGame` function that should return a game object containing:
```

Comment:
The only thing I really wonder about is if this createGame function is asking for too much functionality / state to encourage functional thinking. 

The following bullet points seem a bit like new functions and it feels like we're asking the students to create methods on an object by requesting this functionality within the createGame function. 
- A way to check the game’s board data for win conditions
- A way to detect when a game is a draw (no one has won)
- A way to reset the game’s board to begin a new game

I would love to hear others' opinions on this!

Reply:
Yeah I wondered about that as well. I guess I was picturing those bullet points as separate functions, rather than methods on that object. But that wording will need some adjustments either way. I'll pose this to the group!

Reply:
I've updated the language for those three bullet points to make it clear they are their own methods! Take a look when you have a moment!

Reply:
I love it!